packages/jss/src/DomRenderer.js

```diff
@@ -36,12 +36,13 @@ const getPropertyValue = (cssRule, prop) => {
 const setProperty = (cssRule, prop, value) => {
   try {
     let cssValue = value
+    const important = 'important'
```

Comment:
I would not worry about reusing that string, js VM is doing optimizations that make it not important from perf perspective

Reply:
Done